backend/wmg/data/schemas/cube_schema.py

```diff
@@ -43,9 +41,11 @@
 )
 
 # The cube attributes that comprise the core data stored within the cube.
+filters_numeric = [tiledb.ByteShuffleFilter(), tiledb.ZstdFilter(level=+5)]
```

Comment:
Related to the compression question earlier - what made you choose this combination of byte shuffling and the applying zstandard at level 5?

Reply:
Bruce gave me performance optimization suggestions - these are the filters used in census for numeric attributes.